### PR TITLE
Lookbook

### PR DESCRIPTION
**Pull request by jonrohan**
Adding https://github.com/allmarkedup/lookbook to `/lookbook/` folder. Also connecting with `/css/` workspace included in the vscode codespace config.

I want to make this a starting point, then iterate on migrating stories, as well as improving plugins.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -39,6 +39,10 @@ The category for changes related to documentation, testing and tooling. Also, fo
 
 ### Misc
 
+- Adding [lookbook](https://github.com/allmarkedup/lookbook) dev server to the project
```
Comment (joelhawksley) [inline CHANGELOG.md:42]:
```suggestion
- Adding [lookbook](https://github.com/allmarkedup/lookbook) dev server to the project.
```

File: lookbook/.gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+app/assets/builds/
```
Comment (joelhawksley) [inline lookbook/.gitignore:1]: Delete

File: lookbook/app/helpers/application_helper.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
```
Comment (joelhawksley) [inline lookbook/app/helpers/application_helper.rb:1]: Delete

File: lookbook/app/javascript/controllers/hello_controller.js [hunk 4]
```diff
@@ -0,0 +1,7 @@
+import { Controller } from "@hotwired/stimulus"
```
Comment (joelhawksley) [inline lookbook/app/javascript/controllers/hello_controller.js:1]: Delete

File: lookbook/config/cable.yml [hunk 5]
```diff
@@ -0,0 +1,10 @@
+development:
+  adapter: async
+
+test:
+  adapter: test
+
+production:
+  adapter: redis
+  url: <%= ENV.fetch("REDIS_URL") { "redis://localhost:6379/1" } %>
+  channel_prefix: lookbook_production
```
Comment (joelhawksley) [inline lookbook/config/cable.yml:10]: Let's remove this env for now.

File: lookbook/config/credentials.yml.enc [hunk 6]
```diff
@@ -0,0 +1 @@
+X2ZSbddINz6aRmKSawGzhHpeG2UR2nwG17nIyvEfyVvVL99ebNJxMtbta4hNgU6QOGAYeoLZ7MFSwCTuKgLPSfzlUl3ehoIqJAKcWZbwEMh+Zq+avJgdoFjyzOJLkCQhMjGSsDMqIIv9fPp877XrCfNaR8fik+JFLowxYIbBJsWNXgLtAA6wNqANluBUZ2JDgzgeGn9t9ymUf7+EC0NESILTvOcqyyrhy9OMATqxRwzStXI3WraBxkpyR4/xVX/90NcIUA/Q4nFo6dMRaDO7WmkKbDxdXXnE/JRNoKv+Ben/Xtq+7Mo29Coz7Cu1hSSihF6tIp8JsX2ZaVDuFbLxCAzJqhkfeqtX04vQ+NDa76NOisVNcDuywsLSOs0pQYO4IIaYPVn3DTwPvhF8qoFMMUdaOsSITdabXP6V--gNhmVOO/uGMaVBfi--AS2NIMzGQTW24XQwzQnIag==
```
Comment (joelhawksley) [inline lookbook/config/credentials.yml.enc:1]: Delete

File: lookbook/config/database.yml [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+# Warning: The database defined as "test" will be erased and
+# re-generated from your development database when you run "rake".
+# Do not set this db to the same as development or production.
+test:
+  <<: *default
+  database: db/test.sqlite3
+
+production:
+  <<: *default
+  database: db/production.sqlite3
```
Comment (joelhawksley) [inline lookbook/config/database.yml:25]: Delete

File: lookbook/config/initializers/content_security_policy.rb [hunk 8]
```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
```
Comment (joelhawksley) [inline lookbook/config/initializers/content_security_policy.rb:1]: Delete

File: lookbook/config/initializers/inflections.rb [hunk 9]
```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
```
Comment (joelhawksley) [inline lookbook/config/initializers/inflections.rb:1]: Delete

File: lookbook/config/initializers/permissions_policy.rb [hunk 10]
```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
```
Comment (joelhawksley) [inline lookbook/config/initializers/permissions_policy.rb:1]: Delete

File: lookbook/config/master.key [hunk 11]
```diff
@@ -0,0 +1 @@
+f63d7cfe30f2408c883919416b3c4aad
```
Comment (joelhawksley) [inline lookbook/config/master.key:1]: Delete

File: lookbook/config/storage.yml [hunk 12]
```diff
@@ -0,0 +1,34 @@
+test:
```
Comment (joelhawksley) [inline lookbook/config/storage.yml:1]: Delete

File: lookbook/db/seeds.rb [hunk 13]
```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
```
Comment (joelhawksley) [inline lookbook/db/seeds.rb:1]: Delete

File: lookbook/test/components/previews/counter_component_preview.rb [hunk 14]
```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+# no doc
+class CounterComponentPreview < ViewComponent::Preview
+  # @label Rounded
```
Comment (joelhawksley) [inline lookbook/test/components/previews/counter_component_preview.rb:5]: One of these labels is wrong.

File: lookbook/app/assets/stylesheets/primer.css [hunk 15]
```diff
@@ -0,0 +1,5 @@
+/* If you want to run the workspace with the .vscode/pvc-pcss.code-workspace, */
```
Comment (langermank) [inline lookbook/app/assets/stylesheets/primer.css:1]: Nice! This is a great start ✨

**Review by langermank — APPROVED** [review]

**Pull request merged**